site/app/libraries/PollUtils.php

```diff
@@ -23,13 +23,14 @@ public static function getPollExportData(array $polls): array {
                 $responses[$response] = $poll->getResponseString($response);
             }
             $data[] = [
-                "id" => $poll->getID(),
+                "id" => $poll->getId(),
                 "name" => $poll->getName(),
                 "question" => $poll->getQuestion(),
                 "responses" => $responses,
                 "correct_responses" => $poll->getAnswers(),
                 "release_date" => $poll->getReleaseDate(),
-                "status" => $poll->getStatus()
+                "status" => $poll->getStatus(),
+                "image_file" => $poll->getImagePath()
```

Comment:
Would it make sense to make this be the base64 image string for the export, so that it would make it possible to import the poll and get the same image?

Reply:
Yeah the original plan was to leave the import/export part for images to another time but I think this works pretty well. I'll confirm with barb tomorrow.

Reply:
Talked with Barb -- she wants the export file to remain as human readable as possible so she'd rather ignore the import/export for now and then go back later and have the download be a zip file containing the data and the images themselves.

Reply:
Alright, I think it might make sense to rename this output to `image_path` to better represent what this is, but a small nitpick, and I do not really care.